src/stage1.zig

```diff
@@ -416,7 +416,7 @@ export fn stage2_add_link_lib(
     _ = symbol_name_len;
     _ = symbol_name_ptr;
     const comp = @intToPtr(*Compilation, stage1.userdata);
-    const lib_name = std.ascii.allocLowerString(comp.gpa, lib_name_ptr[0..lib_name_len]) catch return "out of memory";
+    const lib_name = lib_name_ptr[0..lib_name_len];
```

Comment:
This should be duped but since you shouldn't be using stage1 with `--watch` and stage1 is getting deleted soon anyways it's probably fine.